lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java

```diff
@@ -189,7 +190,7 @@ void shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequest() throws Exce
                 .thenReturn(
                         VerifiableCredential.fromValidJwt(
                                 TEST_USER_ID,
-                                "test-cri-id",
+                                ADDRESS,
```

Comment:
This should really be `Cri.CIMIT` - you might need to add it to the enum.

It's a slightly unusual case, in that we don't get them from a CRI and we never store them (but the same is true of TICF). I'm not sure if you'll need to exclude it from any usages of `.values()`.